docs/src/fuzzing/running-fuzzer.md

```diff
@@ -1 +1,10 @@
-# Running the Fuzzer
\ No newline at end of file
+# Running the Fuzzer
+
+Once a fuzzing campaign is set up, you can generally run the fuzzer like:
+
+```sh
+./simics -no-gui --no-win --batch-mode fuzz.simics
+```
+
+At a log level (`tsffs.log-level 2`) of `2` or greater, you'll see statistics of the
+current progress during execution.
```

Comment:
To me, it reads better like this:

At a log level of `2` or greater (ie `tsffs.log-level 2`) , you'll see statistics of the current progress during execution.

Reply:
Agreed!